src/components/actions/scheduler.ts

```diff
@@ -40,11 +48,29 @@ export const triggerScheduledActions = async () => {
   DBConnect.triggerScheduledActions()
 }
 
+const checkFile = async (filePath: string) => {
+  if (await DBConnect.checkIfInFileTable(filePath)) {
+    fs.unlink(filePath, function (err) {
```

Comment:
Please use the existing `deleteFile` function here. (You've imported it already, so you were probably heading in that direction....)